lifecycle-operator/controllers/common/phasehandler_test.go

```diff
@@ -152,7 +152,7 @@ func TestPhaseHandler(t *testing.T) {
 			wantErr: nil,
 			wantObject: &v1alpha3.KeptnAppVersion{
 				Status: v1alpha3.KeptnAppVersionStatus{
-					Status:       apicommon.StateSucceeded,
+					Status:       apicommon.StatePending,
```

Comment:
I would not expect to stay in pending if the state is `apicommon.StateSucceeded` (see line 149)

Reply:
My bad - this is the phase-handler and should not touch the AppVersion status